app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,32 @@
+class CommentsController < ApplicationController
+  before_action :get_post
+  before_action :get_profile_picture
+
+  def create
+    return render json: { view: render_partial_form } if @post && @post.comments.create(comment_params)
+    return render json: { view: render_partial_form } if @profile_picture && @profile_picture.comments.create(comment_params)
```

Comment:
We are returning same thing on both lines, can't we have it in only one line here?